app/status/views.py

```diff
@@ -6,4 +7,11 @@
 
 @status.route('/_status')
 def status_no_db():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    try:
+        return jsonify(status="ok", app_version=utils.get_version_label(),
+                       db_version=utils.get_db_version())
+
+    except SQLAlchemyError:
+        current_app.logger.exception('Cannot connect to database.')
+        return jsonify(status="error", message="Database is down"), 500
```

Comment:
And in future we can add to what the status endpoint returns.  If there are multiple errors (e.g. DB is down and can't connect to search either) - then `_status` should report everything, not just the fist/last failure to be detected.  It won't necessarily be as detailed as this: https://www.digitalmarketplace.service.gov.uk/status - but the more the better I think (within reason).